Makefile

```diff
@@ -77,6 +77,15 @@ configgen:
 	echo "" >> conf/st2.conf.sample
 	. $(VIRTUALENV_DIR)/bin/activate; python ./tools/config_gen.py >> conf/st2.conf.sample;
 
+.PHONY: .generate-openapi-spec
```

Comment:
And yeah, I will probably move it to a new binary and change functionality so it returns a string and doesn't write anything to a file.

This way it can be used in the same and more flexible manner as before.